tests/examples/logging/playground/printstuff/run

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+exec 2>&1
+
+echo sweet
+>&2 echo sweet_error
```

Comment:
That is the weirdest thing I've seen.
It would usually be written as

echo sweet_error >&2